src/test/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/util/TestUtil.java

```diff
@@ -0,0 +1,14 @@
+package com.amazon.opendistro.elasticsearch.performanceanalyzer.util;
+
+import com.amazon.opendistro.elasticsearch.performanceanalyzer.metrics.PerformanceAnalyzerMetrics;
+import com.amazon.opendistro.elasticsearch.performanceanalyzer.reader_writer_shared.Event;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TestUtil {
```

Comment:
Maybe we can rename this class to something more meaningful ?

Reply:
This supposed to be a generate place to put all the utility methods for testing, do you have any suggestion for a better naming?